.gitlab-ci.yml

```diff
@@ -198,6 +199,18 @@ verify:branch:
   script:
     - ASAP_API_URL=$API_URL yarn test:e2e
 
+verify:report:frontend:release:
```

Comment:
I have done this after a successful dev development at which point its already a release.